content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,20 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x, 1.1.x or 3.0.x
+- glibc 2.17 or newer
```

Comment:
also
- zlib (`libz.so.1`)
- libgcc (`libgcc_s.so.1`)

(for these two the exact versions aren’t extremely useful and don’t tell much about the ABI compatibility, or it’s hard to trace what the requirements in terms of versions would be, so I included the [sonames](https://en.m.wikipedia.org/wiki/Soname) instead)

Reply:
This way then?
```suggestion
- glibc 2.17 or newer
- zlib (libz.so.1)
- libgcc (libgcc_s.so.1)
```

Reply:
Note: on `Alpine Linux`, both `libgcc` (the GCC runtime library) and `glibc` aren't needed, right @aqrln?

Reply:
I would also add:
- a statement that informs the user that, most likely, `libgcc`, `zlib` are already installed in their system
- a list of popular Docker images of Linux Distros that we know support the requirements above
- a couple of example of Docker images that don't support the requirements out of the box

Moreover, I have strong suspicion that we actually require a higher `glibc` version on `arm64`. Probably `glibc 2.19` or `glibc 2.24`. I'll confirm that this afternoon

Reply:
We should a general statement about Prisma mostly supporting the `x86_64` and `arm64` CPU architectures, with other ones potentially supported in a "best effort" strategy, with a link to the existing `binaryTargets`

Reply:
@jkomyno

> Note: on Alpine Linux, both libgcc (the GCC runtime library) and glibc aren't needed, right @aqrln?

`glibc` is indeed not needed as musl libc is used instead, but `libgcc` (which Rust uses for stack unwinding on panic) is still required on Alpine and needs to be installed, as it's not present in the vanilla Alpine image. Node.js depends on it too though, so it's already installed in the `node:alpine` image or will be pulled as the dependency when installing Node.js via the package manager.

> Moreover, I have strong suspicion that we actually require a higher glibc version on arm64. Probably glibc 2.19 or glibc 2.24.

We build with glibc 2.24 on ARM64, see https://github.com/prisma/docs/issues/4303#issuecomment-1384701662.

Reply:
Figure out a way to present all this information I would say - but what is suggested here I think already improves it, so I will accept now.